dear_petition/petition/views.py

```diff
@@ -23,6 +25,8 @@ def upload_report(request):
 @login_required
 def view_record(request, pk):
     record = get_object_or_404(CIPRSRecord, pk=pk)
+    if not is_owner(request.user,record.batch) and not request.user.is_superuser:
```

Comment:
Only suggestion is to move ``request.user.is_superuser`` into the ``is_owner`` function for DRY, e.g. ``if not is_owner(request, batch)``